utils/build-windows.bat

```diff
@@ -25,6 +26,10 @@
 
 setlocal enableextensions enabledelayedexpansion
 
+if not defined REPO_SCHEME SET REPO_SCHEME=master

```

Comment:
If `--scheme` is not provided to update-checkout it will automatically use the default branch. 

Can we check the `REPO_SCHME` before adding the flag on line 101? This way we dont have to manage defaults in multiple places.

Reply:
I will try, but Windows batch scripts are not the most flexible thing in the world.